.circleci/config.yml

```diff
@@ -1273,7 +1273,7 @@ jobs:
             export IN_CI=1
 
             # Install sccache
-            sudo curl --retry 3 https://s3.amazonaws.com/ossci-macos/sccache --output /usr/local/bin/sccache
+            sudo curl --retry 3 https://s3.amazonaws.com/ossci-macos/sccache_test --output /usr/local/bin/sccache
```

Comment:
Are you planning to merge with this or you this is for testing purpose?

Reply:
good point, I'll rename the file in S3.